data-fusion/s1_flooding_visualisation/copernicus_browser.js

```diff
@@ -0,0 +1,36 @@
+//VERSION=3
+function setup() {
+    return {
+      input: [{
+          datasource: "S2L2A",
+          bands: ["B08", "B03", "B04"]
+        },
+        {
+          datasource: "S1GRD",
+          bands: ["VV", "VH", "dataMask"]
```

Comment:
The VH band is not accessed, which can be removed

Reply:
Done